src/macfloat.c

```diff
@@ -78,7 +78,7 @@ static void PrintMacfloatToBuf(char *buf, size_t bufsize, Double val, int precis
 */
 void PrintMacfloat(Obj x)
 {
-    Char buf[32];
+    Char buf[1024];
```

Comment:
This change is actually not needed, and no warning is triggered here - at least not for me.

Reply:
You are right, but I decided to be on the safe side I may as well make this bigger too, in case some other compiler ends up producing the same warning, and for consistency.

Reply:
Hrm, that sounds more like voodoo to me than a valid justification... Alas, I don't really care sufficiently to argue this point ;-).